### DISCUSSION TITLE
[Projects Beta] Hide Status boards

### DISCUSSION THREAD

**Discussion by flyingbeat** [community/community#13508] [Projects and Issues]
It should be possible to hide whole Status Boards in the board view and not only hide elements of that status. If i dont want to show elements with status done, this results in an empty board which is still visible...

**Comment by royashbrook** [comment]
Looking for this exact item. I have sprints on a board and wanted to be able to drag items between sprints, but I really only want to see the 'incomplete' items and so I end up with a lot of empty boards and then have to create seperate projects for sprint groups.

I think it should just be a checkbox or something 'hide empty'  or maybe just another filter. In my case, I'd love to have columns by a field, then a way to filter that list in board view. Then i could say show all of the iterations that are say > @previous or even @current or more + items with no sprint.

+1!
